src/traverse.rs

```diff
@@ -1,4 +1,4 @@
-#[derive(Debug)]
+#[derive(Debug, PartialEq)]
```

Comment:
Not sure about this. Why do you need PartialEq? Cant you do check on `.path` instead?

Reply:
Need in the test case src/traverse.rs:397:9